### DISCUSSION TITLE
gp.exe -l -r "ACS ACR1281 1S Dual Reader PICC 0"

### DISCUSSION THREAD

**Discussion by dara7** [martinpaljak/GlobalPlatformPro#329] [Q&A]
Hi guys, i try to install the applet on _Mifare DESFire EV2 4k Card_.
i encounter an issue when run `gp.exe -l -r "ACS ACR1281 1S Dual Reader PICC 0"` or `gp -l`
please help, thanks

Card: **Mifare DESFire EV2 4k Card**
Reader:  **ACR1281U-C1**

**Full log**
Output of  `gp -l`
```
[WARN] GPSession - No mandatory proprietary info present in FCI
Warning: no keys given, using default test key 404142434445464748494A4B4C4D4E4F
Invalid INITIALIZE UPDATE response length: 1
```

**Comment by martinpaljak** [comment]
Mifare desfire is not a javacard

**Reply by dara7** [reply]
ok, so gp doesn't support that card

**Reply by martinpaljak** [reply]
More like "a diesel truck is not supported by this EV charger" but I would not blame it on the connector standard of the tesla charger
